dashboard/dash2.js

```diff
@@ -0,0 +1,134 @@
+// dashboard.js must have been loaded before this one.
+
+"use strict";
+
+// Colors for annotations
+let col_DOWN = "tomato"             // a bit better contrast than "red"
+let col_WORKINGHARD = "deepskyblue" // 75%
+let col_WORKING = "lightskyblue"    // 50%
+let col_COASTING = "lightcyan"      // 25%
+
+// Where to go when clicking on a row, this is opened with ?host=<hostname> from the JSON datum
+let machine_page = `machine-detail.html`
+
+let timeout_minutes = 5;
+
+// At-a-glance data are generated by naicreport, every 5min ideally.
+let fields = [
+    // fqdn or prefix; fqdn is preferred
+    {name: "Host", tag: "hostname"},
+
+    // From the last record on the host.  0=ok, 1=error (maybe more codes later).
+    {name: "CPU\nstatus", tag: "cpu_status"},
+    {name: "GPU\nstatus", tag: "gpu_status"},
+
+    // Unique users in the period.  This will never be greater than jobs; a user can have
+    // several jobs, but not zero, and jobs can only have one user.
+    {name: "Users\n(recent)", tag: "users_recent"},
+    {name: "Users\n(12h)", tag: "users_12h"},
```

Comment:
The `_12h` thing is really not good.  Better would be `longer` maybe, or something slightly indefinite, or based on actual data from the JSON.